schemas/index.js

```diff
@@ -1,162 +1,164 @@
 const Joi = require("@hapi/joi");
 
 const userSchema = Joi.object().keys({
-	username: Joi.string().alphanum(),
-	email: Joi.string()
-		.email({ minDomainSegments: 2 })
-		.required(),
-	first_name: Joi.string()
-		.alphanum()
-		.allow(null),
-	last_name: Joi.string()
-		.alphanum()
-		.allow(null),
-	location: Joi.string()
-		.alphanum()
-		.required(),
-	image: Joi.string().allow(null),
-	banner_image: Joi.string().allow(null),
-	bio: Joi.string().allow(null)
+  username: Joi.string().alphanum(),
+  email: Joi.string()
+    .email({ minDomainSegments: 2 })
+    .required(),
+  first_name: Joi.string()
+    // Allows case-insensitive a-z chars, spaces, apostrophes and dashes
+    .regex(/^[a-z\ \'\-]*$/i)
+    .allow(null),
+  last_name: Joi.string()
+    .regex(/^[a-z\ \'\-]*$/i)
```

Comment:
Good regex for this one, clean and simple

Reply:
I'm getting "first_name" not allowed error on post from Dan's branch, not sure if just something I'm doing wrong, can anyone reproduce with POST to /users?